kong/runloop/handler.lua

```diff
@@ -1092,6 +1136,11 @@ return {
 
   init_worker = {
     before = function()
+      local prefix = kong.configuration.prefix or ngx.config.prefix()
```

Comment:
Please refer to PR #9337, maybe we will introduce a new config var later.

Reply:
@chronolaw I added comment above that line